frontend/src/components/MyNavBar.tsx

```diff
@@ -129,39 +140,48 @@ const MyNavBar = () => {
                     >
                         MyFinance-Manager
                     </Typography>
-                    <Box sx={{flexGrow: 1, display: {xs: 'none', md: 'flex'}}}>
+                    <Box sx={{flexGrow: 1, display: {xs: 'none', md: 'flex'}}}></Box>
 
-                    </Box>
+                    {
+                        props.loggedInUser === "anonymousUser" ? (
+                                <MenuItem onClick={goToLogin}>
 
```

Comment:
Ich kenne das System nicht, mit dem Du arbeitest - aber mich verwirrt ein wenig, dass hier ein MEnuItem ohne vorheriges Menu auftaucht und das gleiche in Zeile 112 (innerhalb eines Menu) steht. Falls das so gehört, hab ich nix gesagt. :)

Reply:
Hast vollkommen recht. ich habe bisschen rumprobiert und einige sachen gelöscht. Morgen werde ich eh das komplette navbar neu erstellen. danke für den hinweis